src/Og.php

```diff
@@ -169,6 +171,31 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
   }
 
   /**
+   * Return TRUE if entity belongs to a group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $group
+   *   The group entity to check.
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to get groups for.
+   * @param array $states
+   *   (optional) Array with the membership states to check the membership.
+   *   Defaults to active memberships.
+   *
+   * @return bool
+   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
+   *   or blocked.
+   */
+  public static function isMember(EntityInterface $group, EntityInterface $entity, $states = [OG_STATE_ACTIVE]) {
```

Comment:
For now we do, yes :) we don't have anywhere else to put them at the
moment. See my other comment about that. We need to see what we've got and
then how it makes sense to split that IMO.
On 7 Dec 2015 17:54, "Daniel Wehner" <notifications@github.com> wrote:

> In src/Og.php
> <https://github.com/amitaibu/og/pull/53#discussion_r46852592>:
>
> > @@ -169,6 +171,31 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
> >    }
> >
> >    /**
> > +   * Return TRUE if entity belongs to a group.
> > +   *
> > +   * @param \Drupal\Core\Entity\EntityInterface $group
> > +   *   The group entity to check.
> > +   * @param \Drupal\Core\Entity\EntityInterface $entity
> > +   *   The entity to get groups for.
> > +   * @param array $states
> > +   *   (optional) Array with the membership states to check the membership.
> > +   *   Defaults to active memberships.
> > +   *
> > +   * @return bool
> > +   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
> > +   *   or blocked.
> > +   */
> > +  public static function isMember(EntityInterface $group, EntityInterface $entity, $states = [OG_STATE_ACTIVE]) {
>
> Do we really need static methods?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/53/files#r46852592>.
>